### ISSUE TITLE
dir_size and cmk2.0

### ISSUE THREAD

**Issue by virus2500** [HeinleinSupport/check_mk_extensions#86]
Hi,

just upgraded to cmk 2. I am using dir_size 4.1 which is in the cmk 2.0 branch but i get the following error when trying to bake the agent.

```
Exception: Error running automation call bake-agents (exit code 2), error: 
Error creating agent for ftp.home.local: Error creating agents: Error creating e85d467675af2e2c/linux_deb agent for ftp.home.local: Error in bakery plugin "dir_size": name 'shutil' is not defined

Traceback (most recent call last):
  File "/omd/sites/main/lib/python3/cmk/base/cee/bakery/agent_bakery.py", line 426, in _execute_pre_17_bakery_plugin
    bake_func(**func_args)
  File "<string>", line 21, in bake_dir_size
NameError: name 'shutil' is not defined
.
.

More information can be found in ~/var/log/web.log
```

Is there something broken with the update or is this plugin not yet compatible?

Thanks in advanced.

**Comment by 47k** [comment]
I think they need to convert the plugin from Python 2 to Python 3. I'm interested in that plugin for CheckMK 2 as well

**Comment by gurubert** [comment]
fixed with commit b03c8a66054df260bbb21781a77b84f1d123b665

**Comment by virus2500** [comment]
thank you
